finalPrep/modZeroHero.js

```diff
@@ -74,11 +161,22 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 
 // - Create 2 instances of your SuperHero class
 
+var hero1 = new SuperHero("Pit Boss", "Can eat anything", 37);
+console.log(hero1);
+
+var hero2 = new SuperHero("House Cat", "Napping", 11);
+console.log(hero2);
 
 // Reflection
 // What parts were most difficult about this exerise?
 
+// The hardest parts of this exercise was how slow I went and how hard the class methods section was for me. //
```

Comment:
✅  Take time to practice building out another class. Either this same exercise, or find Class exercises online (ask on Slack channel if you cannot find resources)